oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -292,55 +288,8 @@ public String getAccount() {
    */
   @Override
   public byte[] sign(byte[] toSign) {
-    BaseEncoding base64 = BaseEncoding.base64();
-    String signature;
-    try {
-      signature = getSignature(base64.encode(toSign));
-    } catch (IOException ex) {
-      throw new SigningException("Failed to sign the provided bytes", ex);
-    }
-    return base64.decode(signature);
-  }
-
-  private String getSignature(String bytes) throws IOException {
-    String signBlobUrl = String.format(SIGN_BLOB_URL_FORMAT, getAccount());
-    GenericUrl genericUrl = new GenericUrl(signBlobUrl);
-
-    GenericData signRequest = new GenericData();
-    signRequest.set("payload", bytes);
-    JsonHttpContent signContent = new JsonHttpContent(OAuth2Utils.JSON_FACTORY, signRequest);
-    HttpRequest request = transportFactory.create().createRequestFactory().buildPostRequest(genericUrl, signContent);
-    Map<String, List<String>> headers = getRequestMetadata();
```

Comment:
Highlighting this change in the `getSignature()` code

Reply:
This was converted to use the `HttpCredentialsAdapter` as an `HttpRequestInitializer` to set the headers rather than doing it manually.